lib/common/client/base.rb

```diff
@@ -54,6 +54,15 @@ def self.configuration(configuration = nil)
         @configuration ||= configuration.instance
       end
 
+      def raise_backend_exception(key, source, error = nil)
```

Comment:
I don't think there's a real plan around it, but how do we feel about keyword arguments ? Nothing you have to do here @LindseySaari just saw the method and was like -- oh, keywords.

Reply:
Are you referring to using an options hash instead or basically running into issues where arguments with the same name as keywords can be dangerous in terms of being overridden?

Reply:
@LindseySaari I mean literal keyword arguments

`def raise_backend_exception(key:, source:, error: nil)`

this has nothing to do directly with your PR! This is just reviewing one method and focusing on one method made me wonder about moving towards keywords. Nothing actionable to do

Reply:
Right, I completely agree!